lua/conform/formatters/trim_newlines.lua

```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/jose-elias-alvarez/null-ls.nvim/blob/main/lua/null-ls/builtins/formatting/trim_newlines.lua",
+    description = "Trim new lines",
```

Comment:
For this and the one below, mention that it's using `awk` in the description and change the link to a landing page for awk. Maybe https://www.gnu.org/software/gawk/manual/gawk.html ?